1.9/usage/cli/command-reference/dcos-config/dcos-config-validate.md

```diff
@@ -0,0 +1,30 @@
+---
+post_title: dcos config validate
+menu_order: 4
+---
+
+# Description
+Validate changes to the configuration file.
+
+# Usage
+
+```bash
+dcos config validate [OPTION]
```

Comment:
there are no options for this command, --help, --info, --version will technically work but they only make sense for a subcommand (ie `dcos config`) or `dcos`